packages/utils/src/rd/index.ts

```diff
@@ -0,0 +1,11 @@
+import { RemoteInitial, RemoteSuccess, RemoteFailure, RemotePending, RemoteData } from '@devexperts/remote-data-ts';
+
+export const initial: RemoteInitial<never, never> = new RemoteInitial<never, never>();
```

Comment:
Isn't it a part of `@devexperts/remote-data-ts`?

Reply:
utils/rd was removed